vanilla-tests/src/main/java/fixtures/xmlservice/Xmls.java

```diff
@@ -379,7 +379,7 @@ public RootWithRefAndNoMeta getComplexTypeRefNoMeta() {
     /**
      * Puts a complex type that has a ref to a complex type with no XML node.
      * 
-     * @param model I am root, and I ref a model with no meta.
+     * @param model The model parameter.
```

Comment:
As long as this is what Swagger defines, this should be fine.

Reply:
The change is because we don't fallback to param's schema's description. So default description is generated..

Reply:
> The change is because we don't fallback to param's schema's description. So default description is generated.

I am asking you to at least check 1 or 2 places in https://github.com/Azure/autorest.testserver/blob/main/swagger, that this assertion is true.